src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/config/EmailConfiguration.kt

```diff
@@ -0,0 +1,70 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.config
+
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.GovNotifyEmailService
+import uk.gov.service.notify.NotificationClient
+
+/**
+ * Email configuration for the service. The sending of all emails is handled by the Gov Notify service.
+ *
+ * If the API key is present then emails will be sent via the Gov Notify service. If the API key is blank/not present
+ * then emails will not be sent.
+ */
+@Configuration
+class EmailConfiguration(
+  @Value("\${notify.api.key:}") private val apiKey: String,
+  @Value("\${notify.templates.prison.transferred:}") private val offenderTransferredPrisonEmailTemplateId: String,
```

Comment:
All of these template id params have all just been taken straight from the whereabouts-api.